src/schema/rules/sidecars/asl.yaml

```diff
@@ -51,12 +53,20 @@ MRIASLTextOnlySliceTiming:
                     final entry in the `SliceTiming` list is the time of acquisition of slice 0.
                     Without this parameter slice time correction will not be possible.
 
-MRIASLTextOnlyFlipAngle:
+MRIASLTextOnlyFlipAngleRec:
+    selectors:
+    - modality == "MRI"
+    - datatype == "perf"
+    - suffix == "asl"
+    - sidecar.LookLocker == false
```

Comment:
Since LookLocker is optional, I don't think `sidecar.LookLocker == false` is sufficient. We either need `sidecar.LookLocker != true`, an or between  `sidecar.LookLocker == false` and `'"LookLocker in sidecar'`, or two separate rules.

Reply:
Ah this relies on a JS idiosyncrasy where `undefined == false` resolves to True. I'll make it more explicit